Xamarin.Forms.Core/ContentPresenter.cs

```diff
@@ -21,9 +21,7 @@ protected override void LayoutChildren(double x, double y, double width, double
 		{
 			for (var i = 0; i < LogicalChildrenInternal.Count; i++)
 			{
-				Element element = LogicalChildrenInternal[i];
```

Comment:
This and `TemplatedView.cs` isn't doing anything useful with `element` so why not get `child` directly? Also, using C# 7 pattern matching now.